composer.json

```diff
@@ -51,13 +51,14 @@
         "twig/twig": "^1.34 || ^2.0"
     },
     "conflict": {
+        "friendsofsymfony/rest-bundle": "<2.1 || >=3.0",
```

Comment:
Do we need an upper limit?

Reply:
It was added on a bundle, and I add it on this one too. We had a discussion about that some time ago, and we decided to keep the upper limit on that other case